common-docs/content/common-theme/front-matter/_index.md

```diff
@@ -0,0 +1,110 @@
++++
+title="Custom Front Matter"
+description="Custom metadata that has special powers in the common theme."
+emoji="🧙"
+menu=["theme"]
++++
+
+## Blocks
+
+```toml
+[[blocks]]
+name="Block name"
+src="path/to/block"
+```
+
+See the [blocks documentation](/common-theme/block-types) for more information.
+
+## Backlog
+
+```toml
+backlog="Module-Name"
+backlog_filter="Week 1"
+```
+
+Only active on on the [Backlog view](/common-theme/pages/backlog). This will show a list of all the issues in the backlog repo for the module. You can filter by any label that exists in the backlog repo.
```

Comment:
```suggestion
Only active on the [Backlog view](/common-theme/pages/backlog). This will show a list of all the issues in the backlog repo for the module. You can filter by any label that exists in the backlog repo.
```